### PR TITLE
[4097] Add service update for HESA integration

### PR DESCRIPTION
**Pull request by gpeng**
### Context

We have enabled the HESA integration.

### Changes proposed in this pull request

Add a service update notifying users.

### Important business

- [ ] Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
- [ ] Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

File: db/service_updates.yml [hunk 1]
```diff
@@ -1,3 +1,10 @@
+- date: '2022-05-09'
+  title: HESA records now visible in Register for April Census update
+  content: |
+    The import from HESA to Register is working normally. This means if you’re a Higher Education Institute (HEI), you can use Register to see data that you have updated and uploaded to HESA. This will allow you to complete your April Census update by 13 May.
+
+    If you identify any further issues with your data, contact [becomingateacher@digital.education.gov.uk](mailto:becomingateacher@digital.education.gov.uk).
```
Comment (MaeveRoseveare) [inline db/service_updates.yml:6]: Can we make the email subject line: April census update - HESA import issues in Register
(So when they click on the link, the email subject is automatically set) - However, not sure you can do that in the news and updates section - if not, don't worry!

**Review by MaeveRoseveare — COMMENTED** [review]
Sorry, I'm being pedantic with the capitalisation!

**Review by MaeveRoseveare — APPROVED** [review]

**Review by AbigailMcP — APPROVED** [review]

File: db/service_updates.yml [hunk 2]
```diff
@@ -1,3 +1,10 @@
+- date: '2022-05-09'
+  title: HESA records now visible in Register for April census update
```
Comment (MaeveRoseveare) [inline db/service_updates.yml:2]:
```suggestion
  title: HESA records now visible in Register for the April census update
```

**Review by MaeveRoseveare — APPROVED** [review]
Final tweaks based on review from Ed.

**Pull request merged**